README.md

```diff
@@ -1 +1,140 @@
-# Content Performance
+# Content Performance Chart
+A small app to visualise the percentage of content people read from each page on [pensionwise.gov.uk]
+against the number of unique pageviews that each page received during a day. Analyses the govspeak 
+content from the [content directory] of the [pension guidance repository] and combines it with data 
+from Google Analytics to produce this [Content Performance Chart].
```

Comment:
If this application is tightly coupled to the Pension Guidance content and analytics maybe we should call that out explicitly when naming the application / repository? e.g. "pension_guidance-performance".

Alternatively we could decouple it so that it can take any markdown content and analytics. The naming currently gives that impression but the implementation does not.